CovidTracker/Pages/Index.cshtml.cs

```diff
@@ -22,9 +22,20 @@ public void OnGet()
         {
             using (var webClient = new WebClient())
             {
-                string jsonString = webClient.DownloadString("https://data.cdc.gov/resource/9mfq-cb36.json");
+                string covid_data = webClient.DownloadString("https://data.cdc.gov/resource/9mfq-cb36.json");
```

Comment:
Gives this a more meaningful name.covid_data gives much more information about the jsonstring than the json string variable.